engine/lib/phx/src/common.rs

```diff
@@ -7,10 +7,10 @@ pub(crate) use c_str;
 
 macro_rules! Fatal {
```

Comment:
Maybe we should just delete `Fatal` and use panic?

Reply:
Yes, I was also thinking about it. Will do it in the follow up PR